internal/store/postgres/migrations/000001_create_providers_table.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS "providers" (
+  "id" uuid DEFAULT uuid_generate_v4(),
+  "type" text,
+  "urn" text,
+  "config" JSONB,
+  "created_at" timestamptz,
+  "updated_at" timestamptz,
+  "deleted_at" timestamptz,
+  PRIMARY KEY ("id")
+);
+
+CREATE INDEX IF NOT EXISTS "idx_providers_deleted_at" ON "providers" ("deleted_at");
+CREATE UNIQUE INDEX IF NOT EXISTS "provider_index" ON "providers" ("type", "urn");
```

Comment:
not sure if we can do this, since these migration scripts are created to replace gorm's auto migration, so the table schemas, indexes, etc. are already present in the db.